ingestion-edge/docker-compose.yml

```diff
@@ -13,6 +13,15 @@ services:
           "projects/test/topics/stub_installer",
           ["GET"]
         ],
+        # ## Proposed routes for account ecosystem telemetry
+        # [
+        #   "/submit/telemetry/<document_id:string>/<suffix:account-ecosystem.*>",
+        #   "projects/test/topics/telemetry-ecosystem"
+        # ],
+        # [
+        #   "/submit/<namespace:string>/<suffix:account-ecosystem.*>",
```

Comment:
These proposed routes use a regex where we match any ping with document type starting with "account-ecosystem", so if we have multiple pings, they could be named "account-ecosystem-deletion-request", etc. and still be routed through the decryptor.

Reply:
The more I think about it, the more I think we're likely to have multiple different pings, so this seems 👍 to me at a high level.

Reply:
As one example of where we might have multiple pings, I think we'll probably want to port the [sync ping](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/data/sync-ping.html) over to use these new identifiers, and there is a *lot* of sync-specific stuff in there that I don't think would make sense to shoehorn into a generic shared schema.